test/components/table/TableFilters.js

```diff
@@ -30,33 +27,34 @@ const columns = [
 ];
 
 test('Renders table filters', t => {
-  const wrapper = shallow(
+  const { container } = render(
     <TableFilters columns={columns} />
   );
 
-  const filters = wrapper.find('.table__filters--filter');
+  const filters = container.querySelectorAll('.table__filters--filter');
```

Comment:
(this comment needs to be prefaced that the team should think about if this is a block/debt or nit): 

These kinds of queries should *probably* be moved to https://testing-library.com/docs/queries/about/#queries 

e.g. https://testing-library.com/docs/marko-testing-library/api/#container-1 

tl;dr querySelector as a query strategy may be vulnerable to DOM changes/ https://medium.com/smallcase-engineering/why-should-we-not-use-document-queryselector-while-testing-react-components-835bbf34169e#7aa1

Reply:
Thanks @Jkovarik. This test was converted by @Nnaga1, but @vindigo and I have used the `querySelector` in some of our tests as well. Some of our tests we were able to use the `screen.findByText` instead of `querySelector`. However, in this particular test I tried using the following method and it results in a failure:  `const filters = screen.getAllByText('.table__filters--filter');`

Perhaps @Nnaga1 might know the correct method to use.

Reply:
Thanks @Nnaga1, this change was in reference to @Jkovarik first comment above: https://github.com/nasa/cumulus-dashboard/pull/1158#discussion_r1889256054

Reply:
@Nnaga1  see the original comment with links.        tl;dr raising the question because enzyme suggests it as a best practice.   See links in original post, and/or https://testing-library.com/docs/marko-testing-library/api/#container-1

Reply:
@Nnaga1  we should probably post-scrum and chat further.

Reply:
Agreed to handle this in a ticket.   Resolving in PR